src/styles/partials/components/_progresstabs.scss

```diff
@@ -0,0 +1,80 @@
+.dg_progress-tabs {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+
+  li {
+    background-color: $gray-lightest;
+    color: $gray;
+    flex: 1 1 auto;
+    font-family: "Montserrat", sans-serif;
```

Comment:
this should use a variable, not sure what it is off the top of my head.

Reply:
hahaha yeah its the default font. Thanks for the catch.